server/services/images/get-signed-url.js

```diff
@@ -0,0 +1,19 @@
+import { v4 as uuid } from 'uuid';
+import { S3, bucket } from './config';
+
+// imageType  | cover | logo | activity
+const getSignedURL = async ({ userId, imageType, fileType, fileName }) => {
+  const key = `${userId}/${imageType}-${uuid()}-${fileName}`;
+  if (!fileType.startsWith('image/')) {
+    throw new Error('must be image/');
+  }
+  const signedURL = await S3.getSignedUrlPromise('putObject', {
+    Bucket: bucket,
+    ContentType: fileType,
+    Key: key,
+  });
+
+  return signedURL;
+};
```

Comment:
You can always just return the promise, I don't think it'd matter, just less code and less wrapping

```suggestion
  return S3.getSignedUrlPromise('putObject', {
    Bucket: bucket,
    ContentType: fileType,
    Key: key,
  });
};
```